src/twisted/conch/test/test_userauth.py

```diff
@@ -27,7 +28,7 @@
 if requireModule("cryptography"):
     from twisted.conch.checkers import SSHProtocolChecker
     from twisted.conch.ssh import keys, transport, userauth
-    from twisted.conch.ssh.common import NS
+    from twisted.conch.ssh.common import MP, NS
```

Comment:
This needs to be a conditional import
```suggestion
    from twisted.conch.ssh.common import MP, NS
    from twisted.conch.test.sk_dummy import DummySK, SKAlgorithm
```